### ISSUE TITLE
А по варианту вроде session scoped

### ISSUE THREAD

**Issue by k1b24** [AlexanderIlin2005/web-lab3#4]
https://github.com/AlexanderIlin2005/web-lab3/blob/051d12c76c93bd648db917ddbaac54a021970471/src/main/java/beans/Model.java#L23C19-L24C1

**Comment by AlexanderIlin2005** [comment]
Да, поставил теперь SessionScoped, хотя в контексте данной лабораторной, видимо, SessionScoped и ApplicationScoped взаимозаменяемы.
